### PR TITLE
Add info about K8s Dashboard operator charm

### PR DESCRIPTION
**Pull request by DomFleischmann**
Documentation of how to deploy k8s dashboard operator.

Signed-off-by: Dominik Fleischmann <dominik.fleischmann@canonical.com>

### CODE REVIEW

File: pages/k8s/cdk-addons.md [hunk 1]
```diff
@@ -63,6 +63,20 @@ juju config kubernetes-master enable-dashboard-addons=false
 juju config kubernetes-master enable-dashboard-addons=true
 ```
 
+For additional control over the Kubernetes Dashboard you can also deploy it into the cluster
```
Comment (evilnick) [inline pages/k8s/cdk-addons.md:66]: What is the additional control?

**Review by evilnick — APPROVED** [review]
Thanks! Two minor things I commented on

File: pages/k8s/cdk-addons.md [hunk 2]
```diff
@@ -63,10 +63,12 @@ juju config kubernetes-master enable-dashboard-addons=false
...
 
-For additional control over the Kubernetes Dashboard you can also deploy it into the cluster
-using the [Kubernetes Dashboard operator bundle](kubernetes-dashboard-bundle). To do so, set
-the `enable-dashboard-addon` [kubernetes-master configuration][] option to `false` and
-deploy the charm into a Kubernetes model on your cluster:
+For additional control over the Kubernetes Dashboard (Different versions,
+authentication methods...) you can also deploy it into the cluster using the
+[Kubernetes Dashboard operator bundle](kubernetes-dashboard-bundle).
+
+To do so, set the `enable-dashboard-addon` [kubernetes-master configuration][]
```
Comment (evilnick) [inline pages/k8s/cdk-addons.md:70]: that config parameter is still 'enable-dashboard-addons' ;)

Comment (DomFleischmann) [reply pages/k8s/cdk-addons.md]: Sorry about that! Should be fixed now.

**Pull request merged**